greenheart/tools/optimization/gc_run_greenheart.py

```diff
@@ -28,35 +33,40 @@ def run_greenheart(config:GreenHeartSimulationConfig, overridden_values=None, ru
     # Otherwise, initialize the WindPark system normally. Get the rank number for parallelization. We only print output files using the root processor.
```

Comment:
Not sure if we want to update this comment since I'm not sure if it's the WindPark system we are parallelizing?

Reply:
Good point. I have updated the comment.